src/entries/popup/handlers/trezor.ts

```diff
@@ -60,8 +56,9 @@ export async function signTransactionFromTrezor(
     }
 
     const nonceHex = BigNumber.from(transaction.nonce).toHexString();
-    const response = await window.TrezorConnect.ethereumSignTransaction({
+    const response = await TrezorConnect.ethereumSignTransaction({
       path,
+      // @ts-expect-error --- unsure why the ts compiler is displaying 'never' for tx type
```

Comment:
This has been tested and works fine. Again, would appreciate input on these type issues stemming from deps.